internal/kibana/synthetics/private_location/read.go

```diff
@@ -25,17 +23,22 @@ func (r *Resource) Read(ctx context.Context, request resource.ReadRequest, respo
 		return
 	}
 
-	label := state.Label.ValueString()
-	namespace := state.SpaceID.ValueString()
-	result, err := kibanaClient.KibanaSynthetics.PrivateLocation.Get(label, namespace)
+	compositeId, dg := synthetics.GetCompositeId(state.ID.ValueString())
```

Comment:
This resource was released as part of 0.11.5, so it's possible customers will have existing resources using the original schema, and attempting to get the composite ID from the old format will fail (it doesn't contain a `/`). 

You could handle it here (i.e use the `Label`/`SpaceID` path if `ID` doesn't contain `/`) or implement a [state upgrade](https://developer.hashicorp.com/terraform/plugin/framework/resources/state-upgrade) which would mean duplicating the schema, but is more explicit.

Reply:
I though `0.11.5` release failed and we could get this change in due-to this: https://buildkite.com/elastic/terraform-provider-elasticstack-release/builds/26

Have I misunderstood it?

Reply:
Done

Reply:
I have implemented the change, but imho it is not necessary. 0.11.5 has import documentation for private locations based on ID, but implementation based on `label`. IMHO it is bug and would not work at all. 

What I try to say, I assume there is no working import functionality for private locations.

Reply:
It's not just imports though, a private location created by 0.11.5 would have an ID in the current format, which would fail to be parsed by the code in this PR.

Reply:
Only for delete, right? There is no update, and I have changed read. And IIUC new code provides new ID during the read, which should replace resource according to schema. Please, correct me, if I miss something. Will fix delete soon.

Reply:
@tobio , I updated delete. Please, advice, what else do I miss.

Reply:
Thanks, read/delete should be enough. Anything else will first read the resource to get the current state so the ID should be updated to the new format before anything else is called anyway.